nipype/interfaces/afni/utils.py

```diff
@@ -25,10 +25,65 @@
     CommandLineInputSpec, CommandLine, Directory, TraitedSpec,
     traits, isdefined, File, InputMultiPath, Undefined, Str)
 from ...external.due import BibTeX
-
+from distutils import spawn
 from .base import (
     AFNICommandBase, AFNICommand, AFNICommandInputSpec, AFNICommandOutputSpec)
 
+class ABoverlapInputSpec(AFNICommandInputSpec):
+    in_file_a = File(
+        desc='input file A',
+        argstr='%s',
+        position=-3,
+        mandatory=True,
+        exists=True,
+        copyfile=False)
+    in_file_b = File(
+        desc='input file B',
+        argstr='%s',
+        position=-2,
+        mandatory=True,
+        exists=True,
+        copyfile=False)
+    out_file = File(
+        desc='collect output to a file',
+        argstr=' |& tee %s',
+        position=-1)
```

Comment:
This may be reimplementing `terminal_output`, though that splits stderr and stdout.